packages/falter-berlin-migration/uci-defaults/freifunk-berlin-01-migration.sh

```diff
@@ -272,6 +272,14 @@ set_ipversion_olsrd6() {
   uci set olsrd6.@olsrd[0].IpVersion=6
 }
 
+remove_deprecated_community_profiles() {
+  # deletes deprecated firmware profiles and copies back only recent profiles
+  # this should ensure to have the most recent version of community profiles everytime
+  log "removing old community profiles"
+  rm /etc/config/profile_*
+  cp /rom/etc/config/profile_* /etc/config/
+}
```

Comment:
Could you please just move this functionality into the migrate_profiles fuction?  The migrate_profiles function is called with every upgrade.

Reply:
solved in new version.

Reply:
please see https://github.com/Freifunk-Spalter/packages/pull/203#issuecomment-903602923

Reply:
oh, completely forgot about that, sorry :D

Reply:
lgtm